app/routes/signin-oidc.js

```diff
@@ -76,6 +76,7 @@ module.exports = [{
         return h.redirect('/claim/visit-review')
       } catch (error) {
         console.error(`Received error with name ${error.name} and message ${error.message}.`)
+        console.log('error', error)
```

Comment:
Is this intentional or it is a debugging log . it it is intentional let put a more descripted label

Reply:
Good spot - I shall remove 😃